rust/src/lib.rs

```diff
@@ -668,6 +668,16 @@ pub mod ffi {
         fn nevra_to_cache_branch(nevra: &CxxString) -> Result<String>;
         fn get_repodata_chksum_repr(pkg: &mut DnfPackage) -> Result<String>;
     }
+
+    // rpmostree-package-variants.h
+    unsafe extern "C++" {
```

Comment:
This syntax with a string looks weird to me but I see: https://cxx.rs/extern-c++.html